netbox/utilities/views.py

```diff
@@ -78,8 +79,9 @@ def has_permission(self):
         if user.has_perms((permission_required, *self.additional_permissions)):
 
             # Update the view's QuerySet to filter only the permitted objects
-            action = resolve_permission(permission_required)[1]
-            self.queryset = self.queryset.restrict(user, action)
+            if isinstance(self.queryset, RestrictedQuerySet):
```

Comment:
We should probably capture this change under a separate issue for visibility.

Reply:
Thinking about this further, we should leave the mixin alone. Instead, we can override `has_permission()` the specific views. We'll need to do this anyway to check for `is_superuser`.

Reply:
Disregard; we can just employ RestrictedQuerySet for the ConfigRevision model.